src/json.hpp

```diff
@@ -29,6 +29,10 @@ SOFTWARE.
 #ifndef NLOHMANN_JSON_HPP
 #define NLOHMANN_JSON_HPP
 
+
+#include "json_forward.hpp" // Some forward declarations are re-used here.
```

Comment:
I do not like breaking the header into two files at this moment. PR #700 proposes a restructuring into several files, but also provides a script to re-create a single header.